### ISSUE TITLE
4.00 Beta2 WebUI should have the Resume / Pause All Transfers buttons like the 3.00 WebUI has

### ISSUE THREAD

**Issue by TorsteinVin** [transmission/transmission#4332]
The old webui from 3.00 had "resume all transfers" and "pause all transfers" buttons in the toolbar, but the new redesigned 4.00 beta2 WebUI are missing these two buttons, and only have a pause/resume button for individual transfers. A workaround now is shift-click all the transfers and press "Pause" or "Resume", but dedicated buttons would make this easier. 

**The previous 3.00 WebUI with the "Resume/Pause All transfers" highlighted with a red box.**
<img width="597" alt="Skjermbilde 2022-12-06 kl  21 28 39" src="https://user-images.githubusercontent.com/77975542/206016211-b7a89168-f0a0-4f63-a413-7bdc64ff37d2.png">

**The new 4.00 beta2 WebUI is missing the "Resume/Pause all transfers" buttons**
<img width="674" alt="Skjermbilde 2022-12-06 kl  21 31 11" src="https://user-images.githubusercontent.com/77975542/206016563-022255b7-7aef-40ad-966b-444c7fcfac3f.png">

**Comment by Deathcow** [comment]
Those buttons were terrible from UIX perspective and should at least require confirmation.

Accidentally clicking one of those with thousands of torrents = panic.

**Comment by nevack** [comment]
> Those buttons were terrible from UIX perspective and should at least require confirmation.

I do agree on that.
Select All -> Pause does the same, and is much less likely to be misclicked/misused

**Comment by killemov** [comment]
> Those buttons were terrible from UIX perspective and should at least require confirmation.
> 
I was thinking about making that configurable for [Shift](https://github.com/killemov/Shift) as well. Starting 20k+ torrents is going to crash my system anyway. However, stopping all torrents should not be a problem. So probably no confirmation on that?

**Comment by GaryElshaw** [comment]
I agree with the comments above and don't think the web client should have those buttons back. Probably a poor analogy, but it's like jamming your bike's front brakes hard. It stops the bike, but doesn't stop you.

**Comment by sfan5** [comment]
AFAICT these do exist now
![grafik](https://user-images.githubusercontent.com/1042418/227740181-f0f3a962-0394-4aab-b9bb-326b514b3b39.png)
